package.json

```diff
@@ -36,6 +36,8 @@
         "@dhis2/d2-i18n": "^1.1.0",
         "@dhis2/prop-types": "^3.1.2",
         "@dhis2/ui": "^9.11.4",
-        "final-form": "^4.19.1"
+        "final-form": "^4.19.1",
+        "react": "~16.14.0",
+        "react-dom": "~16.14.0"
```

Comment:
Couldn't make the app work without specifying a version for React, not sure why it worked before (`yarn install --frozen-lockfile` and `yarn start` doesn't work for me). 

Anyway, I see there is another PR for updating React and other deps, so it's probably fine to do this for now.